runtime/queries/gn/highlights.scm

```diff
@@ -0,0 +1,47 @@
+; Copyright (C) 2021 Will Cassella (github@willcassella.com)
+; 
+; Licensed under the Apache License, Version 2.0 (the "License");
+; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+; 
+;         http://www.apache.org/licenses/LICENSE-2.0
+; 
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS,
+; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+; See the License for the specific language governing permissions and
+; limitations under the License.
+
+(identifier) @variable.builtin
+;((identifier) @variable (#is? @variable definition)) ; This doesn't work
+
+; This is really buggy
+; (call target: (identifier) @_target (#eq? @_target "declare_args")
+;     (block (assignment target: (identifier) @parameter)))
```

Comment:
The only thing I see that could need tuning here is that `parameter` should become `variable.parameter`. I haven't looked at a syntax tree for this though so I might be missing something that needs to change in the pattern

Reply:
I'm just removing this - I'm not quite sure what it was supposed to do and the original author is saying it didn't work